tools/Config.mk

```diff
@@ -523,12 +523,18 @@ ifeq ($(CONFIG_ARCH_COVERAGE),y)
 endif
 
 ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+
+define NEWLINE
+
+
+endef
+
 define CLEAN
 	$(Q) if exist *$(OBJEXT) (del /f /q *$(OBJEXT))
 	$(Q) if exist *$(LIBEXT) (del /f /q *$(LIBEXT))
 	$(Q) if exist *~ (del /f /q *~)
 	$(Q) if exist (del /f /q  .*.swp)
-	$(Q) if exist $(OBJS) (del /f /q $(OBJS))
+	$(foreach OBJ, $(OBJS), $(NEWLINE) $(call DELFILE,$(OBJ)))
```

Comment:
why not call del like before?

Reply:
there's a 'input line is too long' error when use if exist $(OBJS)

Reply:
I mean we can keep del:
```
$(foreach OBJ, $(OBJS), if exist $(OBJ) (del /f /q $(OBJ)))
```

Reply:
without NEWLINE there will be an error: 'there should be no if'
in fact i have tested `$(foreach OBJ, $(OBJS), if exist $(OBJ) (del /f /q $(OBJ)))
` and `$(foreach OBJ, $(OBJS), $(NEWLINE) if exist $(OBJ) (del /f /q $(OBJ)))
` they all emits error in Windows native build